templates/components/dateRange.build.hbs

```diff
@@ -0,0 +1,46 @@
+<template>
+  <label
+    v-if="showPredefinedRanges"
+    :for="name"
+    class="select-proxy-display"
+    :class="{ 'input-focused': isInputFocused , 'readonly': readonly }"
+  >
+    <select
+      class="form-control hidden-select focus-outline"
+      name="predefinedRangesOptions"
+      id="predefinedRangesOptions"
```

Comment:
https://docs.google.com/document/d/1XPlQHPbqp3rGGTekcC05bKtwKDDjrU2RMJ4VIHLXp64/edit#heading=h.yewnp98kurqg

Don't use non-unique IDs in apps because there could be multiple fields and instances of the widget. Personally I don't think `name` or `id` is required here.